README.md

```diff
@@ -2,7 +2,7 @@
 Website that will compare language features side by side.
 
 ## Why would you want this?
-Good question. If there's an aspect of a language you don't know, you can compare a languages you know with a language you do. It's a good way to quickly learn a new language, or use it as a quick reference to remember things by.
```

Comment:
Debates about the effort involved in this PR aside, this line as it is says you can compare a languages you know with a language you know; I imagine the intent was to say `compare a language you don't know with a language you do` or `compare languages you know with a language you don't`.